packages/typespec-ts/src/modular/buildOperations.ts

```diff
@@ -170,6 +143,11 @@ export function buildOperationFiles(
 
     addImportsToFiles(codeModel.runtimeImports, operationGroupFile);
     addImportBySymbol("serializeRecord", operationGroupFile);
+    addImportBySymbol("buildMultiCollection", operationGroupFile);
```

Comment:
does this mean we will generate those helpers regardless if it's applicable or not?

Reply:
Yeah, looks like we do generate regardless for now. Ideally we shouldn't be doing this, but I think it's OK for now. I don't want to touch this area more than I have to in this PR since I know Jose is working in this space with his Binder PR. (@joheredi FYI)

Reply:
Yeah the binder will make sure helpers that are not used are not generated, so I'm okay keeping this for now and my PR on the Binder will address it